cmd/lakefs/cmd/superuser.go

```diff
@@ -24,6 +26,7 @@ var superuserCmd = &cobra.Command{
 		cfg := loadConfig()
 		ctx := cmd.Context()
 		dbPool := db.BuildDatabaseConnection(ctx, cfg.GetDatabaseParams())
+		dbParams := cfg.GetDatabaseParams()
```

Comment:
You can define dbParams before dbPool and then pass it in the BuildDatabaseConnection

Reply:
done